_episodes/03-create.md

```diff
@@ -556,43 +556,54 @@ but it does find the copy in `thesis` that we didn't delete.
 > {: .output}
 >
 > > ## Solution
-> > Jamie needs to move her files `fructose.dat` and `sucrose.dat` to the `analyzed` directory:
-> >
-> > ~~~
-> > $ mv fructose.dat sucrose.dat analyzed
-> > ~~~
+> > ```
+> > mv *.dat analyzed
+> > ```
 > > {: .bash}
+> > The shell will expand *.dat to match all .dat files in the current directory.
+> > The `mv` command then moves the list of .dat files to the "analyzed" directory.
 > {: .solution}
 {: .challenge}
 
 > ## Copy with Multiple Filenames
 >
-> What does `cp` do when given several filenames and a directory name, as in:
+> For this exercise, you can test the commands in the data-shell/data directory.
+>
+> In the example below, what does `cp` do when given several filenames and a directory name?
 >
 > ~~~
+> $ cd ~/data-shell/data
```

Comment:
I would leave out the `cd` here. Many participants will have put the `data-shell` directory in some other location (not `~`).